modules/Preprocessing.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime
+import re
+import string
+from typing import List, Dict
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.corpus import stopwords
+from nltk.tag import pos_tag
+from nltk.tokenize import word_tokenize
+
+
+# lazy load stopwords
+_stopwords = stopwords.words('english')
+_stopwords.extend(['twitter', 'nt'])
+
+
+def _handle_date(date_string: str):
+    """
+    We receive dates from tweets in the format 'yyyy-m-d'.
+    This function returns a datetime object with proper formatting (yyyy-mm-dd)
+    """
+    # return date(*[int(date) for date in date_string.split(',')]) # sorry we didnt get to use you ;(
+    return datetime.strptime(date_string, '%Y,%m,%d').date()
```

Comment:
What is this?

Reply:
what? all of it? #16-22 is already asked above

Reply:
a stopword is a word we cannot use for analysis. This can be discussed further when we go through the project, but for now, here I have printed them out for you:

![image](https://user-images.githubusercontent.com/37186286/81542043-52a11800-9374-11ea-9b96-bb5ba7ef71d3.png)